src/components/v5/common/WidgetBox/WidgetBox.tsx

```diff
@@ -51,16 +51,12 @@ const WidgetBox: FC<WidgetBoxProps> = ({
     </>
   );
 
-  const hoverStyles = 'transition-all sm:hover:border-gray-900';
+  const hoverStyles =
+    'transition-all sm:hover:border-gray-900 sm:hover:text-gray-900 sm:hover:bg-base-white';
```

Comment:
There's 90+ results for `md:hover` vs. ~20 for `sm:hover`. Are you sure the hover styles should be applied from `sm` and not `md`?

Reply:
Thanks for pointing this out. I just moved the styles to another place to tidy up the code :D 

Previously they were directly here:
![image](https://github.com/JoinColony/colonyCDapp/assets/90605528/bfaf7592-e2fc-460e-a99f-3ca6633ad145)

And now everything is here:
![image](https://github.com/JoinColony/colonyCDapp/assets/90605528/afbd0cd3-bb69-4bcb-8f67-70ad830a0f75)

If you are sure that it should be `md` instead of `sm`, I can change it :)